maintainer/conda/MDAnalysis/meta.yaml

```diff
@@ -60,7 +60,7 @@ test:
 
   commands:
     # run the testsuite with 2 processes
-    - python -c 'import MDAnalysisTests; MDAnalysisTests.run(label="full", extra_argv=["-v", "--processes=2", "--process-timeout=120"])'
+    - python -c 'import MDAnalysisTests; MDAnalysisTests.run(label="full", extra_argv=["-v"])'
```

Comment:
I only assign one core the the virtual machine that is used to build the package. So running two test processes makes no sense.